tests/acceptance/crate-test.js

```diff
@@ -182,14 +182,13 @@ test('navigating to the owners page when not logged in', async function(assert)
 test('navigating to the owners page when not an owner', async function(assert) {
     server.loadFixtures();
 
-    this.application.register('service:session-b', Service.extend({
+    this.application.__deprecatedInstance__.register('service:session', Service.extend({
```

Comment:
with the new testing APIs, we will soon be able to replace this with `this.owner.register()` instead. using the private `__deprecatedInstance__` is the official migration path. thanks to @rwjblue for the quick help.